Fabric.Authorization.AccessControl/src/app/services/global/auth.service.ts

```diff
@@ -64,6 +64,9 @@ export class AuthService implements IAuthService {
 
         this.userManager = new UserManager(clientSettings);
 
+        // build out services now..
+        this.servicesService.initialize();
```

Comment:
Need an auth token to hit discovery, but we need to have a UserManager object created before we can get a token to hit DiscoveryService with an oauth token.
So it's in this weird state were we grab the config values, and then build out the services through buildServiceMaps